blog/_posts/2017/2017-01-25-pycon-financial-aid-2017.md

```diff
@@ -0,0 +1,53 @@
+---
+layout: post.html
+title: "Donate to Pyladies @ PyCon 2017"
+tags: [PyCon, FinancialAid]
+author: Dr. Jess Ingrassellino
+author_link: https://twitter.com/jess_ingrass
+---
+
+TL;DR: Donate to PyLadies for PyCon!
+=====
+
+With [PyCon 2017](https://us.pycon.org/2017/) planning in high gear, PyLadies is revving up to raise funds to help women attend the biggest Python conference of the year.
+
+This year, we're hoping to raise at least $20,000 to help women attend this coming PyCon. Here's the breakdown:
+
+OUR NUMBERS
+---
+
+In addition to registration, it will take about $500-1000 per woman in North America to attend PyCon 2017 in Portland, Oregon.
+
+In addition to registration, it will take about $1000-2000 per woman outside of North America to attend PyCon 2017.
+
+WHY PYLADIES? OUR EFFECT ON THE COMMUNITY
+---
+(percentages are roughly estimated based on guessing names-to-gender scripts):
+
+* PyCon 2011 had less than 10% women attendees (speaker % unknown)
+* PyLadies started in late 2011
+* PyCon 2012 had about 11% women attendees, and about the same in speakers. 2013 with about 14 chapters, PyCon had over 20% women attendees and 22% women speakers.
+* For PyCon 2014, we saw that about one-third of attendees and speakers were women.
+
+WHERE DOES YOUR MONEY GO?
+--
+
+* Pyladies has no overhead - we're all volunteers! ;-)
+* Your donations are tax deductible since PyLadies is under the financial enclave of the Python Software Foundation
+
+WHAT YOU WILL GET IN RETURN?
+---
+
+* Unless you explicitly choose to be anonymous (though the online donation site or in private discussions), we will promote and profusely thank you via our public channels ([Twitter](https://twitter.com/pyladies), this blog, and at our PyCon booth)
+* Provide a detailed write-up and financial introspection of how many women we've helped because of this campaign
+* Visibility! It shows the PyCon/Python community that your company/you are serious about women in tech - which can directly benefit sponsors with increased hiring pool of female Pythonistas.
+
+WAYS TO DONATE
+---
+All of the following methods are tax-deductible with funds going to the [Python Software Foundation](https://www.python.org/psf-landing/), a 501(c)3 charitable organization.
```

Comment:
Just one nitpick - mynt renders the `501(c)3` bit as a copyright sign in the middle:

![2017-01-26 at 11 16 am](https://cloud.githubusercontent.com/assets/1191069/22339708/ee6e9d6c-e3b8-11e6-9d27-4ff59bcdedb6.png)

Unfortunately escaping the parens don't help. Could we just change it to `501c3`?